tools/internal/scripts/rush/audit.js

```diff
@@ -45,6 +45,7 @@ const rushCommonDir = path.join(__dirname, "../../../../common/");
     "GHSA-4jqc-8m5r-9rpr", // https://github.com/advisories/GHSA-4jqc-8m5r-9rpr.
     "GHSA-whgm-jr23-g3j9", // https://github.com/advisories/GHSA-whgm-jr23-g3j9.
     "GHSA-r683-j2x4-v87g", // https://github.com/advisories/GHSA-r683-j2x4-v87g
+    "GHSA-wpg7-2c88-r8xv", // https://github.com/advisories/GHSA-wpg7-2c88-r8xv
```

Comment:
Is this still needed? I thought we had fixed this on `master` already.

Reply:
His PR build failed on rush audit due to this (again). But it looks like he hasn't merged master. Let me revert this and pull in master.

Reply:
Let me double check, i thought we needed a new release of keytar for this

Reply:
@calebmshafer I see no relevant commits on master.

Reply:
Ah okay, yea I asked @evelynpreslar-bentley to take a look and as of late yesterday there wasn't a fix.

Reply:
See: https://github.com/atom/node-keytar/issues/438

Reply:
correct, `keytar` is using an old version of `prebuild-install`, which is using a vulnerable version of `simple-get`

I found two packages of ours which use `simple-get` transitively, and both of those go through `keytar` (`full-stack-tests/backend` and `electron-authorization`)

Reply:
@pmconne okay, if that's the case then @aruniverse this is fine to leave in the PR. I just thought it had already been done. Resolving my comment, we'll just need to follow up after the transitive deps are published.